src/AuthService/Program.cs

```diff
@@ -12,16 +12,26 @@ public static void Main(string[] args)
         {
             var configuration = new ConfigurationBuilder()
                 .AddJsonFile("appsettings.json")
-#if DEBUG
```

Comment:
did you ask about it?

Reply:
You can check, it is not necessary

Reply:
I answered this comment in UserService PR. Spartak said this will work, because in Program.cs it's only responsible for logs, so we can delete it.